docs/dle602-a3/quickstart.md

```
@@ -0,0 +1,199 @@
+# DLE602 marker quick start
+
+Three self-contained paths. Pick one; none of them depends on the others.
+
+| Path | Start from | Models available | Needs SemEval data |
+|---|---|---|---|
+| [A](#path-a--run-the-submitted-lightweight-zip) | The submitted ZIP | Five v3 models, plus the complete v2.3.0 workflow | No |
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)